store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/operation/handler/impl/FederatedOperationChainHandlerTest.java

```diff
@@ -99,18 +102,21 @@ public void shouldHandleChainWithoutSpecialFederation() throws OperationExceptio
         final Context context = new Context();
 
         final OperationChain<Iterable<? extends Element>> opChain = new OperationChain.Builder()
-                .first(new GetAllElements.Builder()
-                        // Ensure the elements are returned form the graphs in the right order
-                        .option(KEY_OPERATION_OPTIONS_GRAPH_IDS, GRAPH_IDS)
+                .first(new FederatedOperation.Builder()
```

Comment:
I know this is the new way, but is there a test so that the old way still works? (Ask for a GetAllElements which gets wrapped)?

Reply:
I think...... the standard IT tests covers that.